actions/build-push-docker/action.yml

```diff
@@ -197,6 +201,8 @@ runs:
           ${{
             format('{0},scope={1}', inputs.docker-build-cache-to, runner.arch)
           }}
+        secrets: |
+          GIT_AUTH_TOKEN=${{ inputs.github-token }}
```

Comment:
This allows it to pull go code from private repositories during a build?

Reply:
Yeah, exactly. The Dockerfile will still need to mount the secret for it to be usable though.